.github/workflows/test.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  render:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - run: python -m pip install cookiecutter
+      - run: python -m cookiecutter . --no-input
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: rendered-project
+          path: wagtail-llama-save
+
+  lint:
```

Comment:
In this spirit of delegating everything to pre-commit, I haven't bothered to migrate the eslint build to a job. I'm assuming that eslint will get bundled into pre-commit at a later date and it will run under this step. We can worry about eslint errors then.